storage/bootstrap/types.go

```diff
@@ -66,6 +66,17 @@ type TargetRange struct {
 	RunOptions RunOptions
 }
 
+// ProcessOptions is a set of options for a bootstrap provider.
+type ProcessOptions interface {
```

Comment:
nit: should this `ProcessProviderOptions`?

Reply:
Lol I just renamed it from that @robskillington request